Symfony/CS/Tests/Fixer/Contrib/AlignDoubleArrowFixerTest.php

```diff
@@ -448,6 +448,42 @@ public function provideFixCases()
                 ;
                 ',
             ),
+            array(
+                '<?php
+                $dummy001 = $this->get("doctrine")->getRepository("AppBundle:Entity")->findBy(["server1" => $object], ["addedAt" => "DESC"], 5);
+                $foobar = $this->getDoctrine()->getRepository("AppBundle:Entity")->findBy(["server2" => $object], ["checkedAt" => "desc"], 50);
+                ',
+            ),
+            array(
+                '<?php
+                $dummy001 = $this->get("doctrine")->getRepository("AppBundle:Entity")->findBy(array("server1" => $object), array("addedAt" => "DESC"), 5);
+                $foobar = $this->getDoctrine()->getRepository("AppBundle:Entity")->findBy(array("server2" => $object), array("checkedAt" => "desc"), 50);
+                ',
+            ),
+            array(
+                '<?php
+                $dummy001 = $this->getStrangeDummyManager()->findBy($foo[123]);
+                $foobar = $this->get("manager.dummy")->findBy($foo[123]);
+                ',
+            ),
+            array(
+                '<?php
+                $dummy001 = $this->getStrangeDummyManager()->findBy([1, 2, 3]);
+                $foobar = $this->get("manager.dummy")->findBy([1, 2, 3]);
+                ',
+            ),
+            array(
+                '<?php
+                $dummy001 = $this->getStrangeDummyManager()->findBy((1 + 2));
+                $foobar = $this->get("manager.dummy")->findBy((1 + 2));
+                ',
+            ),
+            array(
+                '<?php
+                $dummy001 = $this->getStrangeDummyManager()->findBy(array(1, 2));
+                $foobar = $this->get("manager.dummy")->findBy(array(1, 2));
+                ',
```

Comment:
all tests you have added have different indentation than current tests. Please have them all be consistence